src/analysis/ml.py

```diff
@@ -300,8 +308,5 @@ def ensemble_network(dataframe: pd.DataFrame, output_file: str):
     """
     row_means = dataframe.mean(axis=1, numeric_only=True).reset_index()
     row_means.columns = ['Edges', 'Frequency']
-    row_means[['Node1', 'Node2']] = row_means['Edges'].str.split(NODE_SEP, expand=True, regex=False)
```

Comment:
Did the output format of the ensemble change here? We are no longer splitting nodes.

Reply:
before we were splitting the nodes, but we had to factor in the directionality into it as we now differ between the two in ml. 

This is the new format: 
Edges	Frequency
A-->B	0.16666666666666666
B-->C	0.16666666666666666
A---B	0.3333333333333333
B---C	0.4166666666666667
B---A	0.08333333333333333
B-->A	0.08333333333333333
C-->B	0.08333333333333333

Reply:
the idea was this based on the comment for the ensemble function: Counts the number of times an edge appears in a set of pathways and divides by the total number of pathways. Edges that appear more frequently across pathways are more likely to be robust,